aws/resource_aws_rds_cluster_test.go

```diff
@@ -46,6 +46,40 @@ func TestAccAWSRDSCluster_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSRDSCluster_crossRegion(t *testing.T) {
+	var v rds.DBCluster
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSClusterConfig(acctest.RandInt()),
```

Comment:
I think we'll also need to create a config which sets up the cross-region replication, `testAccAWSClusterConfig` doesn't do it. Any minimalistic configuration will do in this context, but it should _actually_ setup the replication and test this new behaviour/field - ideally we'd be checking the value of `source_region`, not just whether it's set.

Reply:
This is a little out of my comfort zone.  Unfortunately I don't really have the experience to do what you're asking, I'm not very experienced in writing go.

Reply:
@radeksimko

Reply:
@radeksimko is there anyway we can have this merged?  I'm not the guy to make more complex tests, but this is definitely a needed flag in terraform.